ospd/server.py

```diff
@@ -217,6 +214,9 @@ def start(self, stream_callback: StreamCallbackType):
                 )
             )
 
+        if self.socket_path.exists():
+            os.chmod(str(self.socket_path), self.socket_mode)
```

Comment:
The Path class has chmod method https://docs.python.org/3/library/pathlib.html :-) Not sure why we didn't use that already
```suggestion
            self.socket_path.chmod(self.socket_mode)
```